web3/providers/ipc.py

```diff
@@ -125,8 +125,8 @@ def get_default_ipc_path() -> str:  # type: ignore
 
         base_trinity_path = Path('~').expanduser() / '.local' / 'share' / 'trinity'
         # type ignored b/c ipc_path is already defined as a str above
-        ipc_path = base_trinity_path / 'mainnet' / 'jsonrpc.ipc'  # type: ignore
-        if ipc_path.exists():  # type: ignore
+        ipc_path = base_trinity_path / 'mainnet' / 'jsonrpc.ipc'
+        if ipc_path.exists():
```

Comment:
I think just try leaving these two `# type: ignore`'s in. These two in particular have always been flaky for me.

Reply:
Ah, thanks!